### ISSUE TITLE
clarify instructions in readme.md

### ISSUE THREAD

**Issue by M00NSH0T** [Shopify/shopify_python_api#569]
## Overview/summary

Step 4 in the instructions include a line:

```
access_token = session.request_token(request_params)
```

But it's unclear from a lamen's perspective what request_params should be. Please add some clarification around how to define request_params.

## Motivation

> What inspired this enhancement?

Tring to get up and running quickly with the API...

### Area

- [x] Add any relevant `Area: <area>` labels to this issue

---

## Checklist

- [x] I have described this enhancement in a way that is actionable (if possible)

**Comment by mllemango** [comment]
Hey @M00NSH0T, you raise a really good point. The team will add it to the backlog, in the meantime we always welcome PRs!

**Comment by eporto2021** [comment]
Can someone help a lamen out and clarify exactly what needs to be passed to the request_token() function e.g. an example of 'request_params' ?

**Comment by tamirble** [comment]
I was stuck here too. This is how I solved it.

`code_you_got_from_redirection = "8797f8be6cda4ddcafe1d41aa8d0f859"

session = shopify.Session(shop_url, api_version)

timestamp = time()
request_params = {
    'timestamp': timestamp,
    'code': code_you_got_from_redirection
}

hmac_ = session.calculate_hmac(request_params)
request_params['hmac'] = hmac_
access_token = session.request_token(request_params)
`

**Comment by tamirble** [comment]
I was stuck here too. This is how I solved it.

```
code_you_got_from_redirection = "********"

session = shopify.Session(shop_url, api_version)

timestamp = time()
request_params = {
    'timestamp': timestamp,
    'code': code_you_got_from_redirection
}

hmac_ = session.calculate_hmac(request_params)
request_params['hmac'] = hmac_
access_token = session.request_token(request_params)
```

**Comment by ianjosephwilson** [comment]
@tamirble The code is hard to follow but I think the timestamp and hmac should be provided by shopify via a GET argument.  The library checks the hmac is correct to verify the request was sent by shopify.  There is also a check against the timestamp.
